HW0/P5/P5.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+if __name__ == '__main__':
+  bags = range(1,64)
+  ptime = [np.ceil(np.log(b)) for b in bags]
```

Comment:
You're totally right! As a CS person, I had just assumed the implicit base was 2. Just checked, and it's not :) Thanks for noticing this.